nos/server/http/_service.py

```diff
@@ -87,7 +88,7 @@ def is_video(filename: str) -> bool:
 
 def is_audio(filename: str) -> bool:
     """Check if the given filename is an audio."""
-    return filename.lower().endswith((".mp3", ".wav", ".flac", ".m4a"))
+    return filename.lower().endswith((".mp3", ".mp4", ".wav", ".flac", ".m4a", ".webm"))
```

Comment:
[nit] should probably move this to a `SUPPORTED_AUDIO_EXTENSIONS` const

Reply:
Agreed, will add these on another refactor PR.